package.json

```diff
@@ -9,19 +9,15 @@
         "npm": ">=8.5.0"
     },
     "scripts": {
-        "dev": "supervisor -w Gulpfile.js,webpack.config.js,tsconfig.json supervisor -w Gulpfile.js -x gulp --",
+        "dev": "npx tsx server.ts",
```

Comment:
Reload on change is definitely an important feature for devs. (Is this what you're referring to with the HMR updates needed?)

Reply:
Vite supports live reload by default. However, with HMR (Hot Module Replacement) the specific module which was updated is reloaded, rather than the entire app. [https://webpack.js.org/concepts/hot-module-replacement/](https://github.com/ArnaudBarre/eslint-plugin-react-refresh)

In order to get this working, I have to make updates to a few components. Once this is done, Vite will automatically enable HMR. Related eslint rule: [https://github.com/ArnaudBarre/eslint-plugin-react-refresh](https://github.com/ArnaudBarre/eslint-plugin-react-refresh)